File: day20.py
#!/bin/python3

import sys

n = int(input().strip())
a = list(map(int, input().strip().split(' ')))
# Write Your Code Here
num_swap=0
for i in range(n):
    for j in range(i+1,n):
        if a[i] > a[j]:
            temp=a[j]
            a[j]=a[i]
            a[i]=temp
            num_swap+=1

result1= "Array is sorted in " + str(num_swap) +" swaps."
result2 = "First Element: "+str(a[0])
result3 = "Last Element: " +str(a[n-1])
print(result1)
print(result2)
print(result3)

File: day20.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] a = new int[n];
        for(int a_i=0; a_i < n; a_i++){
            a[a_i] = in.nextInt();
        }
        // Write Your Code Here
        int sum_swap=0;
        for(int i=0; i< n ; i++){
            for(int j=i; j< n ;j++){
                if(a[i]> a[j]){
                  int temp = a[j];
                  a[j] =a[i];
                  a[i] = temp;
                  sum_swap+=1;
                }
            }
        }
        System.out.println("Array is sorted in " + sum_swap +" swaps.");
        System.out.println("First Element: "+ a[0]);
        System.out.println("Last Element: " +a[n-1]);

    }
}
